pkg/rebuild/rebuild/rebuildone.go

```diff
@@ -29,119 +29,109 @@ import (
 )
 
 // RebuildOne runs a rebuild for the given package artifact.
```

Comment:
The signature only tells you that verdict and toUpload are non-nil, want to mention that they contain valid data even when err != nil?